.env.example

```diff
@@ -1,3 +1,8 @@
 PORT=
```

Comment:
Let's make sure that our Environment Variables are looks like this example below.

PORT=9000
NODE_ENV=development 

DATABASE_PORT_DEV=5432
DATABASE_HOST_DEV=localhost
DATABASE_NAME_DEV=e-commerce-ninjas-db
DATABASE_USERNAME_DEV=postgres
DATABASE_PASSWORD_DEV=dbpassword

DATABASE_PORT_PRO=5432
DATABASE_HOST_PRO=online-address.com
DATABASE_NAME_PRO=e-commerce-ninjas-db
DATABASE_USERNAME_PRO=postgres
DATABASE_PASSWORD_PRO=dbpassword

DATABASE_URL_DEV=postgres://postgres:dbpassword@localhost/e-commerce-ninjas-db
DATABASE_URL_PRO=postgres://postgres:dbpassword@online-address.com/e-commerce-ninjas-db


So, when our NODE_ENV changes to ```production```, directly in our src/databases/config/config.ts will use the ```Production Database```.

Reply:
Done